src/components/common/Extensions/UserHub/partials/StakesTab/StakesTab.tsx

```diff
@@ -35,9 +35,26 @@ const StakesTab = () => {
     [stakes],
   );
 
+  const claimedNotificationNumber = useMemo(
+    () => stakes.filter(({ status }) => status === 'claimed').length,
+    [stakes],
+  );
+
+  const updatedTabsItems = useMemo(
+    () =>
+      tabsItems.map((item) => {
+        return item.title === 'Claimable'
```

Comment:
maybe better will be filtering by some id?

Reply:
tabsItems looks like:
```
export const tabsItems: TabItem[] = [
  { id: 0, title: 'All' },
  { id: 1, title: 'Finalized' },
  { id: 2, title: 'Claimable' },
];
```
and are typed by TabsItem[] interface.
How I could do it better?

Reply:
I've added a type prop